backend/corpora/common/corpora_orm.py

```diff
@@ -14,17 +14,46 @@
     String,
     text,
 )
+from sqlalchemy.ext.declarative import DeclarativeMeta
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
 
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from common.corpora_config import CorporaDbConfig
+from .corpora_config import CorporaDbConfig
 
-Base = declarative_base()
-deployment_stage = os.environ["DEPLOYMENT_STAGE"]
 
+class TransformingBase(object):
+    """
+    Add functionality to transform a Base object, and recursively transform its linked entities.
+    """
+
+    RELATIONSHIPS_TO_DICT = True  # Set to True if entities with a relationship to this should be converted to a dict.
```

Comment:
Technically this isn't a global variable so should not be in all-caps.

Reply:
Also, when is this set to false? As it stands today, it never gets set to anything but true? If so, is this class variable actually needed?

Reply:
it was added for future flexibility for Bases that won't be converted to dictionaries. For example, any auth stuff in the DB will not be converted.